### ISSUE TITLE
GetSenv lacks proper error handling

### ISSUE THREAD

**Issue by Zwolf8** [3dsboy08/Synapse-Issues#80]
**Issue Name**: GetSenv lacks proper error handling

**Issue Description**:
So I found this while I was playing around with getsenv if you try to get a script's enviroment when its already finished running it does not error.

`local env = getsenv(game.Players.LocalPlayer.PlayerScripts.gay)
print(#env)
`
the above **does not** cause a error. It is just a empty table because the script that was entered already ran. I believe this needs a proper error because when I was learning the debug library this confused me a lot until I finally figured it out. That's all thanks!

**Describe how to fully reproduce the issue**:

Create a script that will finish running for example just a function that doesn't run

`function empty() print'this prints empty end`
if you try to get the enviroment of this script it will contain nothing

**Other Details**:
tldr; add a error if the localscript's already finished running.

Discord: Zwolf#3762

**Comment by 3dsboy08** [comment]
There is no way to tell if the script is 'finished running' or not.

**Comment by Zwolf8** [comment]
I just wanted this to be added because protosmasher has a error for this  
![2018-06-04_18-44-36](https://user-images.githubusercontent.com/22533350/40950362-4c062ef6-6827-11e8-9fd1-5b9a11908493.png)
